docs/_screencasts/intro/aws.sh

```diff
@@ -0,0 +1,33 @@
+#doitlive speed: 2
+#doitlive prompt: {TTY.CYAN}wash {dir} {r_angle}{TTY.RESET}
```

Comment:
Maybe going back to the old `wash >` might be better. Kind of liked tracking the path so people could see where we're at in the demo, but it is also not the right prompt.